pkg/apis/extensions/install/install.go

```diff
@@ -56,6 +54,15 @@ func init() {
 		return
 	}
 
+	registered.RegisterVersions(availableVersions...)
+
+	if err := registered.EnableVersions(externalVersions...); err != nil {
```

Comment:
EnableVersion just marks it as enabled. The actual enabling is done by addVersionsToScheme() method below, right?
How about renaming EnableVersion to AddEnabledVersion()?

Also, I will suggest calling addVersionsToScheme() first and then call EnableVersion for readability.